src/models.ts

```diff
@@ -37,7 +38,8 @@ export interface MeasurementSystem extends vscode.QuickPickItem{
 }
 
 export interface Space extends vscode.QuickPickItem {
-
+    order_created: number;
+    fileName: string;
```

Comment:
?

Reply:
the fileName state we can probably remove with the changes made to how the calls to getPeirceDB() work, and the order_created is a little bit of additional state that lets me keep track of when a space/interpretation was created so that I can send the API requests in the correct order.

Reply:
That is to say, I have a global variable which will hold a number which is then assigned to this order_created state and incremented when a space/interp is created.

Reply:
^